README.md

```diff
@@ -45,6 +45,19 @@ following versions are ordered from oldest to newest:
 Use the `--version` option on each command line tool to see the software
 version.  An API call reports the software version as a C-style string.
 
+## Releases
+
+Some versions of SPIRV-Tools are tagged as stable releases (see tags on github).
```

Comment:
Suggest linking to the tags URL:   https://github.com/KhronosGroup/SPIRV-Tools/tags